### PR TITLE
QA-684: update the docs with the Mender Server monorepo references

### PR DESCRIPTION
**Pull request by tranchitella**

### CODE REVIEW

File: 07.Server-installation/01.Overview/01.Architecture/docs.md [hunk 1]
```diff
@@ -54,27 +61,47 @@ The architecture below shows the Open Source Mender Server case set up with the
...
+- [Mender Device Authentication Service](https://github.com/mendersoftware/mender-server/tree/main/backend/services/deviceauth/?target=_blank)
+- [Mender Device Configuration Service](https://github.com/mendersoftware/mender-server/tree/main/backend/services/deviceconfig/?target=_blank)
+- [Mender Device Connect Service](https://github.com/mendersoftware/mender-server/tree/main/backend/services/deviceconnect/?target=_blank)
+- [Mender Device Inventory Service](https://github.com/mendersoftware/mender-server/tree/main/backend/services/inventory/?target=_blank)
+- [Mender IoT Manager Service](https://github.com/mendersoftware/mender-server/tree/main/backend/services/iot-manager/?target=_blank)
+- [Mender Reporting Service](https://github.com/mendersoftware/mender-server/tree/main/backend/services/reporting/?target=_blank)
+- [Mender User Administration Service](https://github.com/mendersoftware/mender-server/tree/main/backend/services/useradm/?target=_blank)
+- [Mender Workflows Service](https://github.com/mendersoftware/mender-server/tree/main/backend/services/workflows/?target=_blank)
```
Comment (merlin-northern) [inline 07.Server-installation/01.Overview/01.Architecture/docs.md:79]: device monitor removed on purpose? okey, I read further. ignore.

File: 07.Server-installation/01.Overview/01.Architecture/docs.md [hunk 2]
```diff
@@ -54,27 +61,47 @@ The architecture below shows the Open Source Mender Server case set up with the
...
 - API Gateway based on [Traefik](https://doc.traefik.io/traefik/?target=_blank)
 - [Minio](https://www.minio.io?target=_blank) object storage
 - [NATS.io](https://nats.io?target=_blank) messaging system
 - [OpenSearch](https://www.opensearch.org?target=_blank) storage and search engine
 
-Services are delivered as Docker images, available from the
-official [Mender Docker repository](https://hub.docker.com/r/mendersoftware/?target=_blank).
+The Mender Enterprise Server includes different implementations of some of the services:
```
Comment (merlin-northern) [inline 07.Server-installation/01.Overview/01.Architecture/docs.md:85]: sentece not true when it comes to Monitor service, generate delta, tenantadm, auditlogs.

**Review by merlin-northern — APPROVED** [review]
looks good, I left one comment, and second one I leave here: I think we should add Mender Server to https://github.com/mendersoftware/mender-docs/blob/master/02.Overview/15.Taxonomy/docs.md

File: 08.Server-integration/07.Webhooks/docs.md [hunk 3]
```diff
@@ -89,7 +89,7 @@ The properties included depend on the event type:
 * status change events only include the device id and the new status, 
 * device decommissioning will only include the device id.
 
-More information about payload possibilities are available in the [iot-manager](https://github.com/mendersoftware/iot-manager) management API specification.
+More information about payload possibilities are available in the [iot-manager](https://github.com/mendersoftware/mender-server/tree/main/backend/services/iot-manager/) management API specification.
```
Comment (estenberg) [inline 08.Server-integration/07.Webhooks/docs.md:92]: append ?target=_blank to URL to make it consistent?

**Review by estenberg — APPROVED** [review]
Thanks for cleaning up the Server captialization too. Just a minor note in case you get the chance.

**Pull request merged**